package.json

```diff
@@ -172,6 +172,8 @@
     "lint-staged": "7.3.0",
     "protractor": "5.4.1",
     "react-test-renderer": "16.5.2",
+    "redux-mock-store": "1.5.3",
+    "sass": "1.14.3",
```

Comment:
What's `sass` for?

Reply:
I have no idea. I didn't specifically install `sass` so I thought it came with `redux-mock-store`. Weird, I'm going uninstall it and see if it affects `redux-mock-store`. Sorry about that!